java-checks/src/main/java/org/sonar/java/checks/DuplicateConditionIfElseIfCheck.java

```diff
@@ -37,8 +37,8 @@
 
 @Rule(
   key = DuplicateConditionIfElseIfCheck.RULE_KEY,
-  name = "Conditions in related \"if/else if\" statements should not be duplicated",
-  tags = {"bug", "unused"},
+  name = "Conditions in related \"if/else if\" statements and \"cases\" in a \"(switch|Select Case)\" should not have the same condition",
```

Comment:
RSPEC updated for Java, a switch statement with twice the same case won't compile.